R4ML/R/zzz.R

```diff
@@ -313,7 +313,7 @@ r4ml.session <- function(
   }
   
   # SparkR session init
-  sparkr.init <- function() {
+  sparkr.init <- function(...) {
```

Comment:
I think just have a simple test for testing hiveEnable. so if it works than others will work too.